scripts/remove

```diff
@@ -55,8 +55,24 @@ ynh_remove_nginx_config
 systemctl restart php5-fpm
 systemctl reload nginx
 
+
+#=================================================
+# REMOVE FAIL2BAN CONFIGURATION
+#=================================================
+
+ynh_remove_fail2ban_config
+
+#=================================================
+
+# REMOVE DEDICATED USER
+
+#=================================================
+
+
+
+
```

Comment:
To be removed

Reply:
Why ?

Reply:
Because that's a duplicated comment with blank lines.

Reply:
Oh ok, in my mail it was shown differently, and `ynh_remove_fail2ban_config` was included.
I'll do it right now.